lib/css/base/reset-normalize.less

```diff
@@ -311,7 +311,9 @@ button:-moz-focusring,
  */
 
 fieldset {
-  padding: 0.35em 0.75em 0.625em;
```

Comment:
I'd like to be cautious tweaking either of the resets, as consumers may be depending on those for _other_ styles.

I'd recommend making this change in your component css instead if it is required.

Reply:
Ah, good catch! I [restored this style within the the reset and created fieldset.less](https://github.com/StackExchange/Stacks/pull/1184/commits/c364106cbbc6c591917756f0da7ed5bfc71796bb) which includes the styles previously in `inputs.less`. I chose the standalone `lib/css/base/fieldset.less` file since the style is not specific to any specific component.